nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/canvas/flow-analysis-drawer.jsp

```diff
@@ -0,0 +1,84 @@
+            <div>
+                <div class="flow-analysis-violations-options">
+                    <div class="nf-checkbox checkbox-unchecked" id="show-only-violations"></div>
+                    <span class="nf-checkbox-label show-only-violations-label">Show enforced violations</span>
+                </div>
+                <div class="flow-analysis-warnings-options">
+                    <div class="nf-checkbox checkbox-unchecked" id="show-only-warnings"></div>
+                    <span class="nf-checkbox-label show-only-warnings-label">Show warning violations</span>
+                </div>
+            </div>
+        </div>
+        <div class="flow-analysis-flow-guide-breadcrumb">NiFi Flow</div>
+    </div>
+    <div id="flow-analysis-rules-accordion" class="flow-analysis-rules-accordion">
+
+        <div id="required-rules" class="required-rules">
+            <div>
+                <div>Enforced Rules <span id="required-rule-count" class="required-rule-count"></span></div>
+            </div>
+            <ul id="required-rules-list" class="required-rules-list">
+            </ul>
+        </div>
+
+        <div id="recommended-rules" class="recommended-rules">
+            <div>
+                <div>Warning Rules <span id="recommended-rule-count" class="recommended-rule-count"></span></div>
+            </div>
+            <ul id="recommended-rules-list" class="recommended-rules-list"></ul>
+        </div>
+
+        <div id="rule-violations" class="rule-violations">
+            <div class="rules-violations-header">
+                <div>Enforced Violations <span id="rule-violation-count" class="rule-violation-count"></span></div>
+            </div>
+            <ul id="rule-violations-list" class="rule-violations-list"></ul>
+        </div>
+
+        <div id="rule-warnings" class="rule-warnings">
+            <div class="rules-warnings-header">
+                <div>Warning Violations <span id="rule-warning-count" class="rule-warning-count"></span></div>
+            </div>
+            <ul id="rule-warnings-list" class="rule-warnings-list"></ul>
+        </div>
+
+        <div class="rule-menu" id="rule-menu">
+            <ul>
+                <li class="rule-menu-option" id="rule-menu-view-documentation"><i class="fa fa-info-circle rule-menu-option-icon" aria-hidden="true"></i>View Documentation</li>
+                <li class="rule-menu-option" id="rule-menu-edit-rule"><i class="fa fa-pencil rule-menu-option-icon" aria-hidden="true"></i>Edit Rule</li>
+            </ul>
+        </div>
+
+        <div class="violation-menu" id="violation-menu">
+            <ul>
+                <li class="violation-menu-option" id="violation-menu-more-info"><i class="fa fa-info-circle violation-menu-option-icon" aria-hidden="true"></i>Violation details</li>
+                <li class="violation-menu-option" id="violation-menu-go-to"><i class="fa fa-pencil violation-menu-option-icon" aria-hidden="true"></i>Go to component</li>
```

Comment:
Minor but I noticed inconsistent enabled states on the icons in the context menu.

<img width="352" alt="Screenshot 2024-04-12 at 6 57 04 PM" src="https://github.com/apache/nifi/assets/123395/460a95c2-c84b-46e0-9c1e-c9854efec757">

<img width="349" alt="Screenshot 2024-04-12 at 6 57 15 PM" src="https://github.com/apache/nifi/assets/123395/2ca6f9bd-ef6c-4426-ae4a-0aed7b2a02d9">